bigtable/quickstart/main.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Google LLC
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START bigtable_quickstart]
+// Quickstart is a sample program demonstrating use of the Cloud Bigtable client
+// library to read a row from an existing table.
+package main
+
+import (
+	"flag"
+	"log"
+	"context"
+	"cloud.google.com/go/bigtable"
```

Comment:
group standard library and 3rd party libraries separately: add a newline above the bigtable import

Reply:
done.